content/news/april-20170401.md

```diff
@@ -0,0 +1,45 @@
+The next major OpenRA release is imminent, so you might want to have a look at the upcoming features. If not, just jump directly to the [downloads](/download/) page and grab the latest playtest installer to help us getting feadback.
+
+## TibBucks
+<div style="text-align:center" markdown="1">
+![bucks](/images/news/20170401-user-tib-bucks.png)
+
+Yes, we finally managed to add the highly demanded pay2win features to OpenRA!<br>
+Buy TibBucks, our own virtual ingame currency and get your hands on our bonus content.
+
+![chronoshift icon](/images/news/20170401-chrono-speed-up.png)
+
+Spend TibBucks to paradrop tank armies anywhere<br> on the map or use it to speed your constructions up!
+
+![mammy_hats](/images/news/20170401-mammoth-hats.png)
+
+Customize your units by buying hats for them.
+
+</div>
+
+## New ColorPicker
+
+Our color picker has often been criticized for blocking too many colors and not giving much to select from. Fortunately our fellow comrade '[pulse](http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&p=294293&sid=17a2bc916bb5935e144cfbf0f49adac4#294293)' took a look at it.
+
+<div style="text-align:center" markdown="1">
+![color_picker](/images/news/20170401-color-picker.png)
+
+The old color picker was just not intuitive enough and has been replaced<br> by this non-restrictive, user friendly gem.
+
+</div>
+
+## Brexit news: Britain also leaves the Allies
+
+No worries! We plan to readd them later as own faction ("RTG", the "Real Tough Guys"). They'll be joined by other Commonwealth members/ozeanic nations like Australia and Atlantis.
+
+The United States of America will fill in the empty place in the Allies' rows. They got +200% build speed increase on concrete walls and also have access to Kennels and Guard Dogs. Instead of Tanya they'll get their own hero unit: Donald J. Trump himself! His special abilities are: the best. Just believe us, he has the best and most tremendous abilities.
```

Comment:
s/place/spot/. You'll also get a cookie if you can work "bigly" into that somewhere.